### PR TITLE
State where the flutter.yaml file is located

### PR DESCRIPTION
**Pull request by sethladd**

### CODE REVIEW

**Comment by sethladd** [comment]
cc @qchong

File: assets-and-images.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ and images.
...
 ## Specifying assets
 
-Flutter uses the `flutter.yaml` file to identify assets required by an
-app, as shown in the
-[Introduction to Flutter's Widget Framework](/widgets-intro/).
-A typical `flutter.yaml` file might include something like the following:
+Flutter uses the `flutter.yaml` file, located at the root of your
+project, to identify assets required by an app.
+
+Here is an example of declaring two assets inside `flutter.yaml`:
```
Comment (qchong) [inline assets-and-images.md]: Suggest revising: "Here is an example of how you might declare two assets..."

Comment (sethladd) [inline assets-and-images.md]: Are there other way to declare two assets? (difference between can and might?)

I could also say: "Here is an example of how you can declare two assets" which might be a little bit better?

Comment (qchong) [inline assets-and-images.md]: There are, in fact, two ways the assets might be declared.

assets:
  - assets/my_icon.png
  - assets/background.png

vs.

assets:
  - assets/background.png
  - assets/my_icon.png

So, "might" is more appropriate in this case.

Comment (sethladd) [inline assets-and-images.md]: Oh, commenting on the order. Yeah, we need to state order isn't important.

Comment (sethladd) [inline assets-and-images.md]: ptal

**Comment by qchong** [comment]
LGTM, thanks for the update!

**Pull request merged**